### PR TITLE
Range lists

### PR DESCRIPTION
**Pull request by AnuragSahu11**
Capture2
Capture
image

 *Added lists and 2 types range sliders
*Stacked notification list

### CODE REVIEW

**Review by Tanay-J — APPROVED** [review]

File: components/range-list.css [hunk 1]
```diff
@@ -39,4 +39,8 @@
 }
 .list-square{
     list-style: square;
-}
\ No newline at end of file
+}
+.notification-list-link .card-horizontal{
+    color: var(--font-dark);
+    border: 1px solid black;
```
Comment (stutisk) [inline components/range-list.css]: variable can be used here for black color

**Review by FarhanMobashir — COMMENTED** [review]
All good ✅

**Review by stutisk — COMMENTED** [review]
everything else looks good to me !

File: components/range-list.css (hunk 1 above)
Comment (AnuragSahu11) [reply components/range-list.css]: Yup, fixed it

**Pull request merged**